yevhenii.dolmatov/hash.rb

```diff
@@ -0,0 +1,21 @@
+hash = {
+  'Yes' => 23,
+   'b' => 'travel',
```

Comment:
I notice your code has extra space from this line. In ruby, we use 2 spaces for each indentation.